bedrock/firefox/templates/firefox/welcome/page15/base.html

```diff
@@ -0,0 +1,202 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{% endblock %}
+
+{% block page_image %}{{ static('img/products/vpn/common/social-share.png') }}{% endblock %}
+
+{% block body_class %}welcome-page15{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page15') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-15' %}
+{% set _utm_campaign = 'welcome-15-vpn' %}
+
+{% block experiments %}
+  {% if switch('welcome-15-experiment', ['en-US', 'de', 'fr']) %}
```

Comment:
Need to include en-CA and en-GB here. I think doing just `'en'` covers all three.

Reply:
Good catch!